packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -447,6 +447,7 @@ export function injectIntoGlobalHook(globalObject: any): void {
       globalObject.__REACT_DEVTOOLS_GLOBAL_HOOK__ = hook = {
         renderers: new Map(),
         supportsFiber: true,
+        isDisabled: false,
```

Comment:
nit: We don't need to add this value explicitly. `if (hook.isDisabled)` will fail if it's `undefined` (as it will be if DevTools is the thing that injects the hook first).

Reply:
@bvaughn I'll submit a pull request, and I'll be honored if it's gets merged.